src/m_obj.c

```diff
@@ -919,6 +920,31 @@ int obj_issignaloutlet(const t_object *x, int m)
     "MAINSIGNALIN" has been provided, in which case the object won't
     promote scalars correctly.  Nonetheless we provide it so that at least
     such a badly written object won't crash Pd. */
+
+void obj_sigoutletsetchans(const t_object *x, int m, int nchans)
+{
+    int n;
```

Comment:
Oh, and `n` is not used.

Reply:
`n` is used here:

``` c
for (o2 = x->ob_outlet, n = 0; o2 && m--; o2 = o2->o_next);
```

You normally use `int n = 0`?

Reply:
That is just a left-over from `obj_nsigoutlets` or `obj_sigoutletindex`. Here you just assign 0 to `n` and then do nothing with it :) Just remove it.

BTW, I just saw that there is the same issue in `obj_issignaloutlet` above. You can go ahead and remove that one as well.